dlp/snippets/inspect_content.py

```diff
@@ -1556,6 +1556,320 @@ def inspect_data_to_hybrid_job_trigger(
 # [END dlp_inspect_send_data_to_hybrid_job_trigger]
 
 
+# [START dlp_inspect_gcs_send_to_scc]
+import time  # noqa: F811, E402, I100
+from typing import List  # noqa: F811, E402, I100
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def inspect_gcs_send_to_scc(
+    project: str,
+    bucket: str,
+    info_types: List[str],
+    max_findings: int = 100,
+) -> None:
+    """
+    Uses the Data Loss Prevention API to inspect Google Cloud Storage
+    data and send the results to Google Security Command Center.
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        bucket: The name of the GCS bucket containing the file, as a string.
+        info_types: A list of strings representing infoTypes to inspect for.
+            A full list of infoType categories can be fetched from the API.
+        max_findings: The maximum number of findings to report; 0 = no maximum.
+    """
+    # Instantiate a client.
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Prepare info_types by converting the list of strings into a list of
+    # dictionaries.
+    info_types = [{"name": info_type} for info_type in info_types]
+
+    # Construct the configuration dictionary.
+    inspect_config = {
+        "info_types": info_types,
+        "min_likelihood": google.cloud.dlp_v2.Likelihood.UNLIKELY,
+        "limits": {
+            "max_findings_per_request": max_findings
+        },
+        "include_quote": True,
+    }
+
+    # Construct a cloud_storage_options dictionary with the bucket's URL.
+    url = "gs://{}".format(bucket)
```

Comment:
We prefer f-strings if possible, so `f"gs://{bucket}"`

Reply:
Done